go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/bank-vaults/vault-secrets-webhook
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```

Comment:
We don't need this, because it tells go to download the toolchain and we don't want that.